rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -62,6 +59,11 @@ namespace rerun {
     /// The data will be timestamped automatically based on the `RecordingStream`'s
     /// internal clock.
     class RecordingStream {
+      private:
+        // TODO(grtlr): Ideally we'd expose more of the `EntityPath` struct to the C++ world so
+        //               that we don't have to hardcode this here.
+        inline static const std::string PARTITION_PROPERTIES_ENTITY_PATH = "__partition_properties";
```

Comment:
nit:
```suggestion
       static constexpr const char PARTITION_PROPERTIES_ENTITY_PATH[] = "__partition_properties";
```

_even_ more const, also no startup alloc. That's what we use on string constants in archetypes etc.